packages/grafana-ui/src/components/Logs/LogRowMessage.tsx

```diff
@@ -70,11 +83,15 @@ class UnThemedLogRowMessage extends PureComponent<Props> {
       contextIsOpen,
       showContextToggle,
       wrapLogMessage,
+      prettifyLogMessage,
       onToggleContext,
     } = this.props;
 
     const style = getLogRowStyles(theme, row.logLevel);
     const { entry, hasAnsi, raw } = row;
+    const restructuredEntry = hasAnsi
+      ? restructureLog(raw, prettifyLogMessage)
+      : restructureLog(entry, prettifyLogMessage);
```

Comment:
Just curious, would it break some functionality if we always used `raw` here ? Because if we have ANSI, raw includes ANSI codes. If we don't have ANSI, entry and raw should be the same. https://github.com/grafana/grafana/blob/main/public/app/core/logs_model.ts#L356

Reply:
Yes i changed it to `const restructuredEntry = restructureLog(raw, prettifyLogMessage);`. Should look more clean now!